src/cascadia/TerminalApp/TerminalAppLib.vcxproj

```diff
@@ -439,6 +439,11 @@
       <ResolvedFrom>CppWinRTImplicitlyExpandTargetPlatform</ResolvedFrom>
       <IsSystemReference>True</IsSystemReference>
     </Reference>
+    <Reference Include="$(OpenConsoleDir)src\cascadia\TerminalApp\Microsoft.Management.Deployment.winmd">

```

Comment:
but also, just adding this here, is that enough to actually let our package know what dll these types are implemented in? Or do we need to do something to make sure the winget package (that presumably implements these) is actually in our package graph?

Reply:
_No clue!_ This and [this comment](https://github.com/microsoft/terminal/pull/16848#discussion_r1540928385) are gonna be my next step (figuring out how to actually interact with WinGet).

Reply:
a thought: stage this as two PRs. 
* this first one: plumbing, store the "quick fix" in the buffer, plumb into sxnui. But only store a _blind_ `winget install foo`, don't actually do the package lookup. 
  * probably just hide this behind velocity into canary only for the time being
* a second PR that enlightens the suggestion to include real winget results for `foo`.